installer_scripts/install-yb-voyager

```diff
@@ -94,7 +97,8 @@ centos_main() {
 	$YUM perl perl-DBI perl-App-cpanminus 1>&2
 	$YUM mysql-devel 1>&2
 	$YUM https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm 1>&2 || true
-	$YUM postgresql14-server 1>&2
+	sudo yum -qy module disable postgresql
```

Comment:
Move this line at line 81. We do not want to execute this statement on centos.

Also, will this work on RHEL7?

Reply:
This statement will be required on both Alma Linux and RHEL So I have introduced it before running the `centos_main` function in both the cases. 
I'll test the script on RHEL7 and let you know.

Reply:
The script ran fine after introducing a new case for 'Red Hat Enterprise Linux Server' and not running the `sudo yum -qy module disable postgresql` command.